### PR TITLE
Disable dependabot's auto rebasing

### PR DESCRIPTION
**Pull request by gnakagaw**
### Summary
Dependabot rebase the source branch when it detect a target branch modification by default. It often occur many rebasing and test workflow because We're receiving many PullRequests from dependabot everyday. The storm blocks often our maintenance operations. 

This patch disables the auto-rebasing. After the merging, [we have to send `rebase` message to `@dependabot` manually before a merging.](https://docs.github.com/ja/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/managing-pull-requests-for-dependency-updates#)

Note that we enable [`Require branches to be up to date before merging.` branch protection rule](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-status-checks-before-merging). Thus, it's guaranteed that a PullRequest is updated with `master` branch even if we disable dependabot's auto-merging.

### Checklist
- [-] ~Add a new entry to [CHANGELOG.md](https://github.com/sider/runners/blob/master/CHANGELOG.md) if this change is notable.~ This is not a notable change.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -11,13 +11,15 @@ updates:
...
     labels: ["dependencies", "language: ruby"]
+    rebase-strategy: "disabled"
 
   - package-ecosystem: github-actions
     directory: "/"
     schedule:
       interval: weekly
       time: "09:00"
       timezone: Asia/Tokyo
+      rebase-strategy: "disabled"
```
Comment (radomirbosak) [inline .github/dependabot.yml:22]: This should not be under `schedule`.
Docs: https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#rebase-strategy

Comment (radomirbosak) [reply .github/dependabot.yml:22]:
```suggestion
    rebase-strategy: "disabled"
```

**Review by radomirbosak — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (gnakagaw) [reply .github/dependabot.yml]: @radomirbosak Thanks.

**Review by radomirbosak — APPROVED** [review]
Looks good. :+1: Thank you!

**Pull request merged**